### PR TITLE
New package: Canonical.Ubuntu.2404 version 2404.0.5.0

### LINKED ISSUE #172826: [Update Request]: Ubuntu
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Canonical.Ubuntu

### Package Version

2404.1

### Please describe the changes you would like to see

_No response_

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #172826

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/172850)

### CODE REVIEW

**Comment by Dragon1573** [comment]
Manually validated in _Windows Sandbox_.

image

> [!NOTE]
>
> This package didn't shown in **ARP Entries** after installation. `DisplayVersion` and `Publisher` information can only be found in "Advanced" panel in "Settings > Applications > Installed Applications".

File: manifests/c/Canonical/Ubuntu/2404/2404.0.5.0/Canonical.Ubuntu.2404.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+PackageIdentifier: Canonical.Ubuntu.2404
+PackageVersion: 2404.0.5.0
+PackageLocale: en-US
+Publisher: Canonical Group Limited
+PublisherUrl: https://canonical.com/
+PublisherSupportUrl: https://ubuntu.com/contact-us
+PrivacyUrl: https://ubuntu.com/wsl
+Author: Canonical Ltd. and the OpenSource community
+PackageName: Ubuntu 24.04
```
Comment (Dragon1573) [inline manifests/c/Canonical/Ubuntu/2404/2404.0.5.0/Canonical.Ubuntu.2404.locale.en-US.yaml:12]:
```suggestion
PackageName: Ubuntu 24.04 LTS
```

Should be the same as "Settings" panel.

File: manifests/c/Canonical/Ubuntu/2404/2404.0.5.0/Canonical.Ubuntu.2404.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+InstallerType: appx
+Scope: user
+UpgradeBehavior: install
+PackageFamilyName: CanonicalGroupLimited.Ubuntu24.04LTS_79rhkp1fndgsc
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+ReleaseDate: 2024-04-25
+Installers:
```
Comment (stephengillie) [inline manifests/c/Canonical/Ubuntu/2404/2404.0.5.0/Canonical.Ubuntu.2404.installer.yaml:16]:
```suggestion
Dependencies:
  WindowsFeatures:
    - Microsoft-Windows-Subsystem-Linux 
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image

Edit: My VMs currently have an issue enabling Hyper-V or WSL. (Error 14081)

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Dragon1573** [comment]
@stephengillie @ItzLevvie - This package creates SO MANY https://github.com/microsoft/winget-pkgs/labels/Internal-Error-Dynamic-Scan.

What's wrong with the pipeline? Shall I temporarily switch this pull request as draft and wait for pipelines being recovered?

**Pull request closed without merging**